scripts/scr_bomb_world/scr_bomb_world.gml

```diff
@@ -3,14 +3,14 @@ function scr_bomb_world(star_system, planet_number, bombard_target_faction, bomb
 	var pop_before=0,pop_after=0,reduced_bombard_score=0,strength_reduction=0,txt2="",txt3="",txt4="",max_kill,overkill,roll,kill;
 	var score_before=star_system.p_population[planet_number];
 
-	var txt1="The heavens rumble and thunder as your ship";
+	var txt1="Your cruiser and larger ship";
```

Comment:
I'm thinking - from who's perspective these actions should be described? Serf giving a report? Chapter Master himself witnessing it from the bridge/On the planet that is being bombed? Fleet Master report?
If these can be variable depending on the CMs positioning (on ship that is in bombardment, on the planet that is bombardment, in a different star system), it would be ideal, but I really don't have the knowledge on how to make code recognize these nuances.

2nd bit is theme - overarching theme is grimdark, but how variable (perhaps even deviant) can this be? Can we have traits affecting the one, who is giving the report to the player (or, the CM's traits, if it is from the perspective of CM)? Such as how fearful of The Emperor the reporter is _(in the year of our lord, His armada unleashed...)_, how cynical he is _(Our ships yet again found a desolate rock/unfortunate planet to smash up)_, is he a black humor enjoy-er _(Even I could see the fellows enjoying the flight to the stratosphere. Must be quite the view.)_ and the related possibilities.

But, okay, I have seen choose() command being used, and random(), so, I'll consider it, but I need some instructions on what are we aiming for. Perhaps some text examples on discord.

Reply:
I figured that introduction to what kind of ships - cruisers and Capital ships (frigates and battle barges, perhaps Slaughtersong as well?) are the ones capable to bombard. Although, it is no longer that important, as the screen shows quite clearly what ships can bombard now, I think it is better to have this bit of redundancy.

As for old text of "heavens rumble..." - I think for a proper, professional navy, the player would need to be informed that these procedures take time (especially so, since this can be seen as having effect in game - you can't move fleets around, attack or raid after bombing the world), and, remembering that there are plans to add Surface to Orbit weapons, I might also need to add 1 more TODO, to update this description, once the player becomes under fire by StOs.

While the description of player seeing stuff like Krennic on board the DS-1 when test-firing on Jedha (Oh, it's beautiful) should be the following paragraphs.